packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -491,21 +504,8 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
             // Successful call, make network calls only
             this.firstVersionCall = false;
+            const id = this.initializeFromCachedValue(odspSnapshotCacheValue);
```

Comment:
Feels like API name should be initializeFromSnapshot(), as it does not matter if content came from cache or not